packages/eslint-plugin-stories/src/rules/__tests__/no-new-components-without-story.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { RuleTester } from 'eslint';
+import rule from '../no-new-components-without-story';
+
+const ruleTester = new RuleTester({
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module',
+  },
+});
+
+ruleTester.run('no-mock-db-outside-handler', rule, {
+  valid: [
+    {
+      name: 'a story file',
+      code: 'export default {}',
+      filename: 'foo.stories.tsx',
+    },
+    {
+      name: 'a ts file',
+      code: 'export default {}',
+      filename: 'foo.ts',
+    },
+    {
+      name: 'a spec file',
+      code: 'export default {}',
+      filename: 'foo.spec.tsx',
+    },
+    {
+      name: 'a tsx file without exports',
+      code: 'foo',
+      filename: 'foo.tsx',
+    },
```

Comment:
Is it possible to also test, `"a TSX file with exports"` if we created a simple `foo.stories.tsx` file. For testing purposes, `foo.stories.tsx` could just be an empty file?